README.md

```diff
@@ -1,13 +1,43 @@
 # playwright_framework_tamplate
-## Description
-- webaite
+## About the project
+Automation framework tamplate 
+## Testing strategy
+work in progress..
+## Website description
+Website automationpracice.pl is a clothing store.
 `http://www.automationpractice.pl/index.php`
 ## Install
-`npm install`
+- Install NPM packages
+    ```
+    npm install
+    ```
+- Update     
```

Comment:
Imo it is not necessary, it is unnecessary noise in readme.

Reply:
i deleted it